Examples/StarknetDemo/StarknetDemo/AccountsStore.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  AccountsStore.swift
+//  StarknetDemo
+//
+//  Created by Bartosz Rybarski on 20/01/2023.
+//
+
+import Foundation
+import Starknet
+import BigInt
+
+let erc20: Felt = "0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7"
```

Comment:
Let's maybe name it somehow more meaningful, so if someone reads the code immediately know that this is in fact the contract address.